vulcanai/plotters/visualization.py

```diff
@@ -215,7 +216,7 @@ def _plot_reduction(x_transform, targets, label_map, title, save_path=None):
             "Display environment needed to plot, "
             "or set save_path=path/to/dir")
     else:
-        plt.show(False)
+        plt.show(block=False)
```

Comment:
I believe that this will change the functionality: we want to have plots show up while the model is training, although this does depend heavily on a users' environment. Is there another way to get rid of whatever warning this is related to? https://stackoverflow.com/questions/458209/is-there-a-way-to-detach-matplotlib-plots-so-that-the-computation-can-continue/13361748

Reply:
Repeat for all similar changes below. @rfratila may know more about the intended functionality as I wasn't involved in the writing of this.

Reply:
yes this line is to preserve the plot even once it is no longer training. it is compatible with running python interactively (i.e. in the command line you can run `python -i ...`) to keep the plots up even when the script finishes. More importantly, this is to keep the rest of the script from stalling so it doesn't force you to close the plot in order to continue running the rest of the script.